src/corelibs/U2View/src/ov_sequence/DetView.cpp

```diff
@@ -716,7 +691,7 @@ void DetView::updateVerticalScrollBar() {
         verticalScrollBar->setSliderPosition(0);
     }
 
-    int maximum = 0;
+    int maximum;
```

Comment:
Why did you remove initialization? It's better to initialize variables in any case, even if these variables will be initialized further, because you or somebody may change the code, but won't initialize variable. Please, undo.

Reply:
Use of uninitialized value will break our build.
   QMAKE_CXXFLAGS += -Werror=uninitialized
   QMAKE_CXXFLAGS += -Werror=maybe-uninitialized

The artificial '0' value will hide real bugs: if we forget to assign a real value to it nobody will detect this issue.
Not using the artificial value at all will make a compiler to detect the problem and break Commit build.